linode/vpc/resource_test.go

```diff
@@ -0,0 +1,207 @@
+//go:build integration
+
+package vpc_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/linode/linodego"
+	"github.com/linode/terraform-provider-linode/linode/acceptance"
+	"github.com/linode/terraform-provider-linode/linode/helper"
+	"github.com/linode/terraform-provider-linode/linode/vpc/tmpl"
+)
+
+var testRegion string
+
+func init() {
+	resource.AddTestSweepers("linode_vpc", &resource.Sweeper{
+		Name: "linode_vpc",
+		F:    sweep,
+	})
+
+	testRegion = "us-east" // VPC only available in certain regions currently
+}
+
+func sweep(prefix string) error {
+	client, err := acceptance.GetTestClient("https://api.dev.linode.com")
```

Comment:
Can we assume it's working with beta and use the get capability function? I think we can use alpha url to do local tests but it would be nicer to hide it in the public code base.